mountpoint-s3/src/fs.rs

```diff
@@ -833,6 +906,10 @@ where
                 metrics::decrement_gauge!("fs.current_handles", 1.0, "type" => "read");
                 Ok(())
             }
+            FileHandleType::Uninitialized { .. } => {
+                metrics::decrement_gauge!("fs.current_handles", 1.0, "type" => "uninitialized");
+                Ok(())
+            }
```

Comment:
I guess we're effectively pretending it was a read if it was never read/written?

Reply:
Not quite sure why it's read?

Reply:
If we were pretending it was a write, we'd have to upload a 0-byte object here (which seems like the wrong thing to do!).

Reply:
Right, if we have to pick one it should be a read.